diff --git a/dist/index.js b/dist/index.js
index badee5803289701abe5c68072e2c5b4df5eae731..c6963673f00676d5081b5dc24d7d1b141a570c97 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -270,8 +270,8 @@ var ETransactionVersion3 = {
 };
 
 // src/types/api/index.ts
-var RPCSPEC07 = __toESM(require("starknet-types-07"));
-__reExport(api_exports, require("starknet-types-07"));
+var RPCSPEC07 = __toESM(require("@starknet-io/types-js"));
+__reExport(api_exports, require("@starknet-io/types-js"));
 
 // src/utils/encode.ts
 var encode_exports = {};
@@ -675,7 +675,7 @@ var OutsideExecutionVersion = /* @__PURE__ */ ((OutsideExecutionVersion2) => {
 })(OutsideExecutionVersion || {});
 
 // src/types/typedData.ts
-var import_starknet_types_07 = require("starknet-types-07");
+var import_starknet_types_07 = require("@starknet-io/types-js");
 
 // src/utils/batch/index.ts
 var BatchClient = class {
diff --git a/dist/index.mjs b/dist/index.mjs
index cd1c7da0b13a5e529db6ced2cba37f1c2eb11f60..2295f57861caaa18f80bd38448c8706b36b29ac8 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -147,8 +147,8 @@ var ETransactionVersion3 = {
 
 // src/types/api/index.ts
 __reExport(api_exports, starknet_types_07_star);
-import * as RPCSPEC07 from "starknet-types-07";
-import * as starknet_types_07_star from "starknet-types-07";
+import * as RPCSPEC07 from "@starknet-io/types-js";
+import * as starknet_types_07_star from "@starknet-io/types-js";
 
 // src/utils/encode.ts
 var encode_exports = {};
@@ -554,7 +554,7 @@ var OutsideExecutionVersion = /* @__PURE__ */ ((OutsideExecutionVersion2) => {
 // src/types/typedData.ts
 import {
   TypedDataRevision
-} from "starknet-types-07";
+} from "@starknet-io/types-js";
 
 // src/utils/batch/index.ts
 var BatchClient = class {
